File: Python/백준/Bronze/30017. 치즈버거 만들기/치즈버거 만들기.py
a,b = map(int,input().split())
cnt = 0
while(True):
    if(cnt%2==0):
        if(a!=0):
            a-=1
        else:
            break
    else:
        if(b!=0):
            b-=1
        else:
            break
    cnt+=1
if cnt%2==0:
    print(cnt-1)
else:
    print(cnt)

File: C#/백준/Bronze/30017. 치즈버거 만들기/치즈버거 만들기.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
            if(a[0]<=a[1]){
                Console.WriteLine(a[0]*2-1);
            }
            else{
                Console.WriteLine(a[1]*2+1);
            }
        }
    }
}